docs/griptape-framework/data/src/loaders_save.py

```diff
@@ -0,0 +1,10 @@
+from griptape.drivers import LocalFileManagerDriver
+from griptape.loaders import TextLoader
+
+loader = TextLoader(file_manager_driver=LocalFileManagerDriver())
+
+data = loader.load("tests/resources/test.txt")
+
+data.value = data.value.replace("foo", "bar")
+
+loader.save("tests/resources/test.txt", data)
```

Comment:
This is starting to remind me of pathlib

Reply:
🤷